tests/std/tests/GH_005236_collate_facet/test.cpp

```diff
@@ -19,6 +19,83 @@
 
 using namespace std;
 
+// GH-5210 "std::collate<_Elem>::do_transform() should behave appropriately when _LStrxfrm() fails"
+void test_gh_5210() {
+#ifndef SKIP_COLLATE_TRANSFORM_TESTS
+    {
+        locale utf8_locale("en_US.utf8");

```

Comment:
We conventionally use locale names with dashes ("en-US.UTF-8" is used elsewhere). It used to make a difference on old OSes, which didn't understand underscores. I don't *think* we're using those systems in our internal harness anymore, but consistency seems worthwhile to keep. Occurs below (e.g. this test has pre-existing "de-DE_phoneb" but this PR is adding "de_DE_phoneb").